site/docs/red-team/agents.md

```diff
@@ -4,6 +4,8 @@ sidebar_position: 10001
 
 # How to red team LLM Agents
 
+> **Tip:** For agent-based systems, we recommend using the [Promptfoo Target Scanning Agent (PTSA)](/docs/red-team/ptsa) to automatically discover the agent's purpose, capabilities, and constraints. PTSA runs automatically as part of the red team scan and helps generate more targeted adversarial tests.
```

Comment:
why do you recommend it? You should update the part of this file that mentions the purpose below instead.